src/one_dragon/envs/git_service.py

```
@@ -196,63 +230,126 @@ def stage_progress_callback(progress: float, message: str) -> None:
-
-            # 检查本地是否存在该分支且有提交历史
-            depth = 1  # 默认深度为1
-            local_ref = f'refs/heads/{branch_name}'
-            if local_ref in repo.references:
-                # 分支存在，检查是否有提交历史
-                branch_target = repo.references[local_ref].target
-                if branch_target is not None:
-                    # 有提交历史，使用增量拉取
-                    depth = 0
+            primary_remote = self._ensure_remote()
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)